src/contributing/structure.md

```diff
@@ -0,0 +1,21 @@
+# Documentation Structure
+
+The directory structure is meant to feel comfortable for those familiar with the data platform:
```

Comment:
The directory structure should feel comfortable to anyone who is familiar with the data platform: